### PR TITLE
Remove TypeScript apps, packages and pipelines

### PR DESCRIPTION
**Pull request by lachieh**
This PR removes the TypeScript apps and packages from the main repo, given that they now have a home in the [TypeScript language monorepo](https://github.com/wasmCloud/typescript).

The two commits break down the actions taken.
- I have updated wash to search for washboard versions in the TS repo
- I have removed the code related to the washboard and the packages used for it.

It does not make any changes to the examples yet, as the TS repo does not yet include the workflows for publishing the typescript examples, though the examples themselves have been copied into the TS repo (tracking issue: wasmCloud/typescript#2)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -49,5 +49,5 @@ crates/wash-cli @wasmCloud/wash-maintainers
 crates/wash-lib @wasmCloud/wash-maintainers
 examples @wasmCloud/examples-maintainers
 vendor/capability-providers @wasmCloud/capability-provider-maintainers
-typescript @wasmCloud/washboard-maintainers
+examples/typescript @wasmCloud/typescript-maintainers
```
Comment (lachieh) [inline .github/CODEOWNERS:52]: This team [exists](https://github.com/orgs/wasmCloud/teams/typescript-maintainers), but may not have access to the repo.

File: flake.nix [hunk 2]
```diff
@@ -41,52 +41,48 @@
...
+          "nix"
+          "OWNERS"
+          "performance.md"
+          "README.md"
+          "RELEASE.md"
+          "RELEASE_RUNBOOK.md"
+          "ROADMAP.md"
+          "rust-toolchain.toml"
+          "SECURITY.md"
+        ];
```
Comment (lachieh) [inline flake.nix:85]: This basically reverts what was added when wash was integrated into the repo.

File: crates/wash-cli/src/ui/mod.rs [hunk 3]
```diff
@@ -68,6 +68,9 @@ async fn ensure_washboard(version: &str, base_dir: PathBuf) -> Result<PathBuf> {
 
 async fn download_washboard(version: &str, install_dir: &PathBuf) -> Result<()> {
     let urls = vec![
+        format!(
+            "https://github.com/wasmCloud/typescript/releases/download/washboard-ui%40{version}/washboard.tar.gz"
+        ),
```
Comment (brooksmtownsend) [inline crates/wash-cli/src/ui/mod.rs:73]: When should we remove the old URLs, or should we just keep them indefinitely?

**Review by brooksmtownsend — APPROVED** [review]

File: crates/wash-cli/src/ui/mod.rs (hunk 3 above)
Comment (lachieh) [reply crates/wash-cli/src/ui/mod.rs:73]: Probably on the next release? It doesn't hurt to leave them, aside from a few extra bytes.

**Pull request merged**